Jenkinsfile

```diff
@@ -294,6 +294,56 @@ pipeline{
 						}
 					}
 				}
+				stage ('Worker build - Aarch64 Windows guest') {
```

Comment:
Why run this as a new worker invokation? Are you going to be providing a new system for doing that? It would be better to just run the steps after the usual integration test steps on aarch64.

Reply:
ACK, moved into the existing aarch64 stage, squashed and rebased to latest main.

Thanks